src/Microsoft.AspNetCore.OData/Query/Expressions/QueryBinder.cs

```diff
@@ -1487,42 +1546,42 @@ private Expression BindDynamicPropertyAccessExpression(SingleValueOpenPropertyAc
     /// <summary>
     /// Creates an expression for retrieving a dynamic property from the dynamic properties container property.
     /// </summary>
-    /// <param name="dynamicPropertiesContainerExpr">The dynamic properties container property access expression.</param>
+    /// <param name="containerPropertyAccessExpr">The dynamic properties container property access expression.</param>
```

Comment:
Since you are passing `containerPropertyAccessExpr` and not `dynamicPropertiesContainerExpr`, should you also change the doc string for this parameter as well
```suggestion
    /// <param name="containerPropertyAccessExpr">The  properties container property access expression.</param>
```

Reply:
Maybe `The container properties property access expression`. Otherwise, `The properties container properties access expression` would sound inaccurate given the explanation I have provided here https://github.com/OData/AspNetCoreOData/pull/1322#discussion_r1832318661
"The dynamic properties container property access expression" is captures the definition accurately too